.travis.yml

```diff
@@ -1,17 +1,37 @@
 language: python
-python:
-    - "3.6"
-    - "2.7"
-sudo: false
 
 matrix:
   include:
     - os: osx
       language: generic
-# install dependecy
+      sudo: false
```

Comment:
I'm not sure if this is a huge deal or not but in Appveyor we are testing for py3.4 and py3.5 as well.